projects/ardin_mb/vector_field.cc

```diff
@@ -0,0 +1,121 @@
+#include "vector_field.h"
```

Comment:
Another (possibly easier but less flexible) option to doing this in OpenGL would be to use matplotlib-cpp's ``quiver()`` function

Reply:
Yeah - the problem is that I want to overlay the result on the OpenGL map view.

I guess theoretically we could draw that whole view using matplotlib but I can see that turning into a shit show.

Reply:
That's true. Plus I suspect you'd have to add your own wrapper function for rendering the triangles anyway.